login-utils/login.c

```diff
@@ -862,7 +862,6 @@ static const char *loginpam_get_prompt(struct login_context *cxt)
 {
 	const char *host;
 	char *prompt, *dflt_prompt = _("login: ");
```

Comment:
Probably set `prompt` to NULL to make some static analyzers happy.

Reply:
Added as requested